ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py

```diff
@@ -43,3 +43,38 @@ def test_paddle_layer_norm(
         bias=offset[0],
         epsilon=eps,
     )
+
+
+# normalize
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.normalize",
+    dtype_and_x_and_axis=helpers.arrays_and_axes(
+        available_dtypes=helpers.get_dtypes("float"),
+        num=1,
+        return_dtype=True,
+        force_int_axis=True,
+    ),
+    p=st.floats(min_value=0.1, max_value=2),
+)
+def test_paddle_normalize(
+    *,
+    dtype_and_x_and_axis,
+    p,
+    test_flags,
+    frontend,
+    backend_fw,
+    on_device,
+    fn_tree,
+):
+    dtype, x, _ = dtype_and_x_and_axis
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        backend_to_test=backend_fw,
+        frontend=frontend,
+        test_flags=test_flags,
+        on_device=on_device,
+        fn_tree=fn_tree,
+        x=x[0],
+        p=p,
+        axis=0,
```

Comment:
I just noticed this right now. But this is quite restrictive. `axis`. Could you kindly add value generation for the `axis` parameter as well following the domain defined in the paddle docs [here](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/normalize_en.html#:~:text=axis%20(int%2C%20optional)%20%E2%80%93%20The%20axis%20on%20which%20to%20apply%20normalization.%20If%20axis%20%3C%200%2C%20the%20dimension%20to%20normalization%20is%20x.ndim%20%2B%20axis.%20%2D1%20is%20the%20last%20dimension.)